E3db/Classes/Crypto.swift

```diff
@@ -188,10 +188,16 @@ extension Crypto {
 extension Crypto {
 
     static func signature(doc: Signable, signingKey: Sign.SecretKey) -> String? {
-        let message = try? Crypto.base64UrlDecoded(string: doc.serialized())
-        return sodium.sign.signature(message: message!, secretKey: signingKey)?.base64UrlEncodedString()
+        var message: [UInt8] = Array("".utf8)
```

Comment:
considering the time you spent fighting this func, it could be worth adding a comment about what kind of data `doc` is expected to be.

Reply:
💯